.builds/openbsd.yml

```
@@ -0,0 +1,34 @@
+image: openbsd/latest
+packages:
+- sqlite3
+- node
+- boehm-gc
+- pcre
+- sfml
+- sdl2
+sources:
+- https://github.com/nim-lang/Nim
+environment:
+  CC: /usr/bin/clang
+tasks:
+- setup: |
+    cd Nim
+    git clone --depth 1 -q https://github.com/nim-lang/csources.git
+    cd csources
+    sh build.sh
```

```suggestion
    gmake -C csources -j $(nproc)
```

Using GNU make will enable parallelization of csources building, which helps a lot.

---

Is `gmake` installed by default? If not, I'll need to add it to the packages list.